src/Management/Assistants/UrlAssistant.php

```diff
@@ -56,7 +56,7 @@ public function fields(): Fields
             InputField::make('url-slugs')
                 ->validation(
                     [
-                        'url-slugs' => ['array', 'min:1', new UniqueUrlSlugRule(($this->model && $this->model->exists) ? $this->model : null),],
+                        'url-slugs' => ['array', 'min:1', new UniqueUrlSlugRule($this->model, ($this->model->exists) ? $this->model : null),],
```

Comment:
De haakjes rond $this->model->exists mogen weg of zouden rond de hele ternary check mogen staan.
Misschien ook de trailing comma weglaten?